keps/sig-api-machinery/4006-transition-spdy-to-websockets/README.md

```diff
@@ -228,10 +228,10 @@ between `kubectl` and the API Server.
 
 2. Transition the bi-directional streaming protocol from SPDY/3.1 to WebSockets
 for `kubectl port-forward` for the communication leg between `kubectl` and the API
-Server (alpha in v1.29).
+Server.
 
 3. Extend the WebSockets communication leg from the API Server to Kubelet
-*before* the current leg goes GA (probably in v1.30). After this extension, WebSockets
+*before* the current leg goes GA (probably in v1.31). After this extension, WebSockets
```

Comment:
drop the version estimate from here

include this in the GA graduation criteria below

Reply:
Removed estimate. Added GA graduation criteria for extending WebSockets from API Server to Kubelet.